### PR TITLE
8516 removed copy The actions(s) in PAS Racial Equity Section

### PR DESCRIPTION
**Pull request by horatiorosa**
### Summary
updated copy line 15 in `equitable-development-reporting.hbs` : The actions(s) from "The action(s) I am applying for are ULURP actions with an acquisition or disposition of land for:

#### Tasks/Bug Numbers
 - Related [AB#8374](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/8374)
Screen Shot 2022-05-05 at 1 37 42 PM

### CODE REVIEW

File: client/app/components/packages/equitable-development-reporting.hbs [hunk 1]
```diff
@@ -12,7 +12,7 @@
       </p>
 
       <p>
-        The action(s) I am applying for are ULURP actions with an acquisition or disposition of land for:
+        I am applying for are ULURP actions with an acquisition or disposition of land for:
```
Comment (godfreyyeung) [inline client/app/components/packages/equitable-development-reporting.hbs:15]: Why does this copy look so off.... "I am appling for are " just sounds like something is missing?

Comment (horatiorosa) [reply client/app/components/packages/equitable-development-reporting.hbs:15]: It is awkward. `are` probably should be dropped or leave `The action(s)` in. Aline wanted to drop that but perhaps didn't review the rest of the copy.

Comment (godfreyyeung) [reply client/app/components/packages/equitable-development-reporting.hbs:15]: the previous made more sense..

Comment (godfreyyeung) [reply client/app/components/packages/equitable-development-reporting.hbs:15]: gotcha. 

> It is awkward. `are` probably should be dropped or leave `The action(s)` in. Aline wanted to drop that but perhaps didn't review the rest of the copy.

Gotcha. Yeah i think you're right...

Comment (godfreyyeung) [reply client/app/components/packages/equitable-development-reporting.hbs:15]: They probably meant  " I am applying for ULURP actions with an acquisition or disposition of land for:" ?

**Review by godfreyyeung — APPROVED** [review]

**Pull request merged**